app/Models/Collection.php

```diff
@@ -117,6 +117,10 @@ public function bannerUpdatedAt(): ?Carbon
     {
         $bannerUpdatedAt = $this->extra_attributes->get('banner_updated_at');
 
+        if ($bannerUpdatedAt === null) {
+            return null;
+        }
+
         return Carbon::parse($bannerUpdatedAt);
```

Comment:
remove parsing, let the callee handle that and just return the value as-is from `$this->extra_attributes->get('banner_updated_at');`